chrome-extension/content.js

```diff
@@ -70,40 +65,46 @@ function closeSidebar() {
 function createSidebar(content) {
   sidebar.innerHTML = '';
   sidebar.style.display = 'block';
-  const panelHeading = document.createElement('div');
-  panelHeading.classList.add('message-header');
-  panelHeading.innerText = 'Lobster Ice Cream';
+
+  const panelHeading = customCreateElement('div', ['message-header'], 'Lobster Ice Cream');
   //TODO: Include icon in panelhead, update looks
 
-  const close = document.createElement('a');
-  close.classList.add('delete');
+  const close = customCreateElement('a', ['delete']);
   close.onclick = closeSidebar;
   panelHeading.appendChild(close);
   sidebar.appendChild(panelHeading);
 
   const form = document.createElement('form');
-  const highlightLabel = document.createElement('label');
-  highlightLabel.classList.add('label', 'mt-2');
-  highlightLabel.innerText = 'Highlighted Text';
+
+  const highlightLabel = customCreateElement('label', ['label', 'mt-2'], 'Highlighted Text');
   form.appendChild(highlightLabel);
 
-  const highlightTextarea = document.createElement('textarea');
-  highlightTextarea.classList.add('textarea');
+  const highlightTextarea = customCreateElement('textarea', ['textarea']);
   form.appendChild(highlightTextarea);
   highlightTextarea.value = content;
 
-  const noteTextarea = document.createElement('textarea');
-  noteTextarea.classList.add('textarea', 'mt-6');
+  const noteTextarea = customCreateElement('textarea', ['textarea', 'mt-6']);
```

Comment:
Consider naming this commentTextarea, since it populates the comment property. This matches its id in your later PR

Reply:
yep! will do